### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by thisisdano**
See comments for more about what I did here.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+16
```
Comment (thisisdano) [inline .nvmrc:1]: Update node to latest LTS

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+3.0.2
```
Comment (thisisdano) [inline .ruby-version:1]: Update Ruby to latest LTS

File: Gemfile [hunk 3]
```diff
@@ -11,3 +11,7 @@ group :jekyll_plugins do
 end
 
 gem 'html-proofer', '~> 3.16'
+
+gem "webrick", "~> 1.7"
+
+gem "rexml", "~> 3.2"
```
Comment (thisisdano) [inline Gemfile:17]: Add additional gems not included by default in Ruby 3

File: _scss/components/_header.scss [hunk 4]
```diff
@@ -87,7 +87,7 @@
   @include at-media-max("desktop") {
     border-color: color("base");
   }
-  @include u-height("10");
+  @include u-height(10);
```
Comment (thisisdano) [inline _scss/components/_header.scss:90]: Fix quoted numbers throughout to fix compile errors

File: gulpfile.js [hunk 5]
```diff
@@ -7,7 +7,7 @@ var pkg = require("./node_modules/uswds/package.json");
 var postcss = require("gulp-postcss");
 var rename = require("gulp-rename");
 var replace = require("gulp-replace");
-var sass = require("gulp-sass");
+var sass = require("gulp-dart-sass");
```
Comment (thisisdano) [inline gulpfile.js:10]: Update to current Sass compiler

File: package.json [hunk 6]
```diff
@@ -4,26 +4,28 @@
...
-    "gulp-sourcemaps": "^2.6.5",
+    "gulp-sourcemaps": "^3.0.0",
     "gulplog": "^1.0.0",
     "js-yaml": "^3.13.1",
     "lodash.assign": "^4.2.0",
     "pa11y-ci": "^2.3.0",
     "path": "^0.12.7",
+    "sass": "^1.44.0",
     "smooth-scroll": "^16.1.0",
+    "underscore": "^1.13.1",
```
Comment (thisisdano) [inline package.json:28]: Update underscore to latest version

**Comment by thisisdano** [comment]
Currently a visual regression in the Glossary. Fixing now.

**Comment by thisisdano** [comment]
OK, checks out. Updated to `underscore.js 1.13.1`

**Comment by thisisdano** [comment]
I see a Circle pass at https://app.circleci.com/pipelines/github/GSA/digitalgov-pra/335/workflows/e4170f90-ce5f-48a8-a8e9-794e532c5704/jobs/336 — not sure why it's not being reported here

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**